auth-server/server.py

```diff
@@ -54,20 +54,30 @@ def connect_to_database():
     print("Connected to database!")
 
 
+def usersSerializer(user):
+    return {
+        "username": user.username,
+        "password": user.password,
```

Comment:
I wouldn't return users passwords in clear. Do sth like
```suggestion
"password": "*" * len(user.password)
```

Reply:
Done.